src/raymath.h

```diff
@@ -990,22 +990,22 @@ RMAPI Vector3 Vector3Unproject(Vector3 source, Matrix projection, Matrix view)
     float invDet = 1.0f/(b00*b11 - b01*b10 + b02*b09 + b03*b08 - b04*b07 + b05*b06);
 
     Matrix matViewProjInv = {
-        (a11*b11 - a12*b10 + a13*b09)*invDet,
+        ( a11*b11 - a12*b10 + a13*b09)*invDet,
```

Comment:
I prefer to avoid this kind of alignments based on additional spaces. One space is ok despite lines are not aligned.